main.cpp

```diff
@@ -270,6 +305,18 @@ float Headphones::Microphone::trackInputLevel(bool useGainToDecibelsTransformati
     return inputLevel;
 }
 
+void Headphones::Microphone::increaseInput(float gain)
+{
+    if (inputGain < gain)
+        inputGain = gain;
+    
+    while (inputGain <= 1.0f)
+        {
+            inputGain += 0.1f;
+            std::cout << "Input gain: " << inputGain << std::endl;
+        }
```

Comment:
FIXME: indentation
curly braces should line up with the keywords on the line above
- [ ] revise